profile/README.md

```diff
@@ -26,7 +25,10 @@ and platform-portable, written in Go
 * [Plugins](https://github.com/orgs/arcalot/repositories?q=arcaflow-plugin-), written in
 various languages and delivered as Linux containers or modules
 * [Workflow definitions](https://github.com/arcalot/arcaflow-workflows), written in
-YAML, tieing together plugins and their data
+YAML, tying together plugins and their data
```

Comment:
Or maybe ", providing the engine with instructions to connect built-in functions and plugins to produce machine-readable output data"? Or, to borrow a phrase from below, creating a "pipeline of actions". (Although we call them "steps", and "pipeline" suggests a linearity without decisions, and ...)

Reply:
The quick descriptions here need to be super-short to get the point across in a tl;dr kind of way.  I'm not in love with the use of the word "pipeline," but I don't want to keep saying "workflow" over and over again either.

Reply:
How about "sequencing the execution of plugins and directing the flow of data between them".